test/e2e/network/fixture.go

```diff
@@ -84,13 +85,15 @@ func (t *TestFixture) BuildServiceSpec() *v1.Service {
 	return service
 }
 
-// CreateRC creates a replication controller and records it for cleanup.
-func (t *TestFixture) CreateRC(rc *v1.ReplicationController) (*v1.ReplicationController, error) {
-	rc, err := t.Client.CoreV1().ReplicationControllers(t.Namespace).Create(context.TODO(), rc, metav1.CreateOptions{})
+// CreateDeployment creates a deployment and records it for cleanup.
+func (t *TestFixture) CreateDeployment(deployment *appsv1.Deployment) (*appsv1.Deployment, error) {
```

Comment:
one commit can be just change this method CreateRC by CreateDeployment and all test associated?
or we can just do this in on PR and the submit additional PRs for the other changes, but the most important change here is exactly this one